### PR TITLE
Add license headers to SDK files

### PR DESCRIPTION
**Pull request by Cloudwu1**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
Add license to all .h, .m, .java and .js files.

Example for license:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

## Related PRs or issues

List related PRs and other issues.

## Misc

Some files are generated from React-Native and add app center code, I'm not sure if they need to add Microsoft license. currently I have add it to them.

### CODE REVIEW

File: BrownfieldTestApp/ios/BrownfieldTestApp/AppDelegate.m [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 /**
```
Comment (dhei) [inline BrownfieldTestApp/ios/BrownfieldTestApp/AppDelegate.m]: We should remove Facebook's copyright header from all test app code.

File: TestApp34/android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -1,2 +1,2 @@
-#Mon Oct 30 16:27:59 MSK 2017
 connection.project.dir=
+eclipse.preferences.version=1
```
Comment (dhei) [inline TestApp34/android/.settings/org.eclipse.buildship.core.prefs]: What is file and why do we need to check it into Git?

Comment (dhei) [reply TestApp34/android/.settings/org.eclipse.buildship.core.prefs]: If we don't need it, let's remove it.

Comment (guperrot) [reply TestApp34/android/.settings/org.eclipse.buildship.core.prefs]: Fixed in 12f875e

**Pull request merged**